modules/database/util.bal

```diff
@@ -40,61 +40,3 @@ isolated function checkAllValues((any|error)[] values) returns sql:Value[]|error
     }
     return result;
 }
-
-# Custom class that allows concatenating raw templates to dynamically build sql queries.
-#
-# **Deprecated**, as there is a function in ballerina that allows this concatenation.
-public class ConcatQuery {
```

Comment:
The complete file can be removed. The other functions were support functions for this class.